### PR TITLE
update some dockerfiles to support power

### PR DESCRIPTION
**Pull request by dreamryx**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/katib/673)

### CODE REVIEW

File: cmd/suggestion/random/v1alpha1/Dockerfile [hunk 1]
```diff
@@ -2,7 +2,12 @@ FROM golang:alpine AS build-env
 # The GOPATH in the image is /go.
 ADD . /go/src/github.com/kubeflow/katib
 WORKDIR /go/src/github.com/kubeflow/katib/cmd/suggestion/random
-RUN go build -o random ./v1alpha1
+RUN if [ "$(uname -m)" = "ppc64le" ]; then \
+        apk --update add gcc musl-dev  && \
```
Comment (hougangliu) [inline cmd/suggestion/random/v1alpha1/Dockerfile]: @dreamryx good job.
do you miss katib/cmd/suggestion for v1alpha2?

File: cmd/ui/v1alpha2/Dockerfile [hunk 2]
```diff
@@ -2,7 +2,12 @@ FROM golang:alpine AS build-env
 # The GOPATH in the image is /go.
```
Comment (andreyvelich) [inline cmd/ui/v1alpha2/Dockerfile]: Can you rebase with new Dockerfile for the Katib UI in v1alpha2, please ?

File: cmd/suggestion/random/v1alpha1/Dockerfile (hunk 1 above)
Comment (dreamryx) [reply cmd/suggestion/random/v1alpha1/Dockerfile]: OK. I will re-commit docker file for v2.

File: cmd/ui/v1alpha2/Dockerfile (hunk 2 above)
Comment (dreamryx) [reply cmd/ui/v1alpha2/Dockerfile]: OK. I will commit the v2 docker files. Thanks.

Comment (dreamryx) [reply cmd/ui/v1alpha2/Dockerfile]: OK.

File: cmd/manager-rest/v1alpha2/Dockerfile [hunk 3]
```diff
@@ -2,7 +2,12 @@ FROM golang:alpine AS build-env
 # The GOPATH in the image is /go.
 ADD . /go/src/github.com/kubeflow/katib
 WORKDIR /go/src/github.com/kubeflow/katib/cmd/manager-rest
-RUN go build -o katib-manager-rest ./v1alpha2
+RUN if [ "$(uname -m)" = "ppc64le" ]; then \
+        apk --update add gcc musl-dev  && \
+        go build -o vizier-manager-rest ./v1alpha2; \
+    else \
+        go build -o vizier-manager-rest ./v1alpha2; \
```
Comment (hougangliu) [inline cmd/manager-rest/v1alpha2/Dockerfile]: s/vizier-manager-rest/katib-manager-rest

**Pull request merged**